include/language/en_us.lang.php

```diff
@@ -3927,3 +3928,36 @@
 );
 
 $app_list_strings['moduleList']['Spots'] = 'Spots';
+
+$app_list_strings['moduleList']['AOBH_BusinessHours'] = 'Business Hours';
+$app_list_strings['business_hours_list']['0'] = '12am';
+$app_list_strings['business_hours_list']['1'] = '1am';
+$app_list_strings['business_hours_list']['2'] = '2am';
+$app_list_strings['business_hours_list']['3'] = '3am';
+$app_list_strings['business_hours_list']['4'] = '4am';
+$app_list_strings['business_hours_list']['5'] = '5am';
+$app_list_strings['business_hours_list']['6'] = '6am';
+$app_list_strings['business_hours_list']['7'] = '7am';
+$app_list_strings['business_hours_list']['8'] = '8am';
+$app_list_strings['business_hours_list']['9'] = '9am';
+$app_list_strings['business_hours_list']['10'] = '10am';
+$app_list_strings['business_hours_list']['11'] = '11am';
+$app_list_strings['business_hours_list']['12'] = '12pm';
+$app_list_strings['business_hours_list']['13'] = '1pm';
+$app_list_strings['business_hours_list']['14'] = '2pm';
+$app_list_strings['business_hours_list']['15'] = '3pm';
+$app_list_strings['business_hours_list']['16'] = '4pm';
+$app_list_strings['business_hours_list']['17'] = '5pm';
+$app_list_strings['business_hours_list']['18'] = '6pm';
+$app_list_strings['business_hours_list']['19'] = '7pm';
+$app_list_strings['business_hours_list']['20'] = '8pm';
+$app_list_strings['business_hours_list']['21'] = '9pm';
+$app_list_strings['business_hours_list']['22'] = '10pm';
+$app_list_strings['business_hours_list']['23'] = '11pm';
+$app_list_strings['day_list']['Monday'] = 'Monday';
```

Comment:
these names should not be translated, otherwise it will create issues.

regarding numeric or textual, Jim is the best person to answer this question as this is his package that i have integrated in core.

________________________________
From: Ashley Nicolson <notifications@github.com>
Sent: Thursday, July 21, 2016 11:34 AM
To: salesagility/SuiteCRM
Cc: Haris Raheem; Mention
Subject: Re: [salesagility/SuiteCRM] US 1.9 (#1768)


In include/language/en_us.lang.php<https://github.com/salesagility/SuiteCRM/pull/1768#discussion_r71689419>:

> +$app_list_strings['business_hours_list']['9'] = '9am';
> +$app_list_strings['business_hours_list']['10'] = '10am';
> +$app_list_strings['business_hours_list']['11'] = '11am';
> +$app_list_strings['business_hours_list']['12'] = '12pm';
> +$app_list_strings['business_hours_list']['13'] = '1pm';
> +$app_list_strings['business_hours_list']['14'] = '2pm';
> +$app_list_strings['business_hours_list']['15'] = '3pm';
> +$app_list_strings['business_hours_list']['16'] = '4pm';
> +$app_list_strings['business_hours_list']['17'] = '5pm';
> +$app_list_strings['business_hours_list']['18'] = '6pm';
> +$app_list_strings['business_hours_list']['19'] = '7pm';
> +$app_list_strings['business_hours_list']['20'] = '8pm';
> +$app_list_strings['business_hours_list']['21'] = '9pm';
> +$app_list_strings['business_hours_list']['22'] = '10pm';
> +$app_list_strings['business_hours_list']['23'] = '11pm';
> +$app_list_strings['day_list']['Monday'] = 'Monday';


Would this cause any issues being translatable in the later calculations? Should these be numeric rather than textual.

-
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/salesagility/SuiteCRM/pull/1768/files/fd2e95d057d200bdf3055088acf61e4fd49f4f59..592b89c19b92334533d4b6493046dc282c12e1c6#r71689419>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ARj4l-R7jnQwH1oUq4ocNejP4uPzKQLgks5qX1ldgaJpZM4JQtrL>.